mysql-test/mroonga/storage/r/max_execution_time.result

```diff
@@ -4,12 +4,12 @@ CREATE TABLE diaries(
 title TEXT
 ) DEFAULT CHARSET=utf8mb4;
 INSERT INTO diaries VALUES ('Hello');
-SELECT /*+ MAX_EXECUTION_TIME(1) */ sleep(1), title FROM diaries;
+SELECT /*+ MAX_EXECUTION_TIME(1) */ title FROM diaries WHERE sleep(1) OR title IS NOT NULL;
 ERROR HY000: Query execution was interrupted, maximum statement execution time exceeded
-SELECT sleep(1), title FROM diaries;
-sleep(1)	title
-0	Hello
-SELECT /*+ MAX_EXECUTION_TIME(10000) */ sleep(1), title FROM diaries;
-sleep(1)	title
-0	Hello
+SELECT title FROM diaries WHERE sleep(1) OR title IS NOT NULL;
+title
+Hello
+SELECT /*+ MAX_EXECUTION_TIME(10000) */ title FROM diaries WHERE sleep(1) OR title IS NOT NULL;
```

Comment:
`OR title IS NOT NULL` is for getting values of the title column.
If we specify only `sleep()`, there is no match records.

Reply:
Could you add it to `.test` as a comment?